install/installer/rest_installer

```diff
@@ -108,13 +108,6 @@ mv $MY_SQROOT/sqenvcom.temp $MY_SQROOT/sqenvcom.sh
 cd $REST_DIR/conf
 
 echo "***INFO: modifying $REST_DIR/conf/rest-site.xml"
-# get zookeeper quorum
-
-lineNumber=$(grep -n "zookeeper\.quorum" /etc/hbase/conf/hbase-site.xml | sed 's/\:.*//')
-lineNumber=$((lineNumber+1))
-
-ZOOKEEPER_NODES=`sed "$lineNumber!d" /etc/hbase/conf/hbase-site.xml | sed 's/\/value.*//' | sed 's/.*>//' | sed 's/.$//'`
-
 # add zookeeper quorum property to end of configuration
 rm rest-site.temp 2>/dev/null
 cat rest-site.xml | sed -e "s@</configuration>@  <property>\n    <name>rest.zookeeper.quorum</name>\n    <value>$ZOOKEEPER_NODES</value>\n  </property>\n </configuration>@" > rest-site.temp
```

Comment:
Should we source in the traf config file ( as done in dcs_installer ) to pickup $ZOOKEEPER_NODES ? or is it available at this point ?
     source $TRAF_CONFIG